### DISCUSSION TITLE
N64 Decompiling with Ghidra - Retro Reversing (Reverse Engineering)

### DISCUSSION THREAD

**Discussion by giscus** [RetroReversing/retroReversing#181] [Post Comments]
# N64 Decompiling with Ghidra - Retro Reversing (Reverse Engineering)

RetroReversing, the home of retro game reverse engineering. Get resources, tutorials, and tools for analyzing video games and join the community today!

https://www.retroreversing.com/n64-decompiling

**Comment by svinsmoke212** [comment]
The second hyperlink in your references is broken. https://assemblergames.com/threads/reverse-engineering-n64-signature-files-for-ida-pro-radare2.70138/
